pages/common/php-yii.md

```diff
@@ -0,0 +1,15 @@
+# php yii
+
+> Yii Framework's command line interface.
+
+- Display a list of all available commands:
+
+`php yii {{help}}`
+
+- Start PHP's built-in web server for the current Yii application:
```

Comment:
What's the difference between this and `php -a`?

Reply:
This command serves the application while bootstrapping the framework and using appropriate configuration values where necessary.

Reply:
Ah, ok